### PR TITLE
Automated PR for IBM-pibs

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/IBM-pibs.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <body>
+      <p>This source code has been made available to you by IBM on an AS-IS basis. Anyone receiving this source is
+         licensed under IBM copyrights to use it in any way he or she deems fit, including copying it,
+         modifying it, compiling it, and redistributing it either with or without modifications. No license
+         under IBM patents or patent applications is to be implied by the copyright license.</p>
+      <p>Any user of this software should understand that IBM cannot provide technical support for this software
+         and will not be responsible for any consequences resulting from the use of this software.</p>
+      <p>Any person who transfers this source code or any derivative work must include the IBM copyright notice,
+         this paragraph, and the preceding two paragraphs in the transferred software.</p>
+      <p>COPYRIGHT I B M CORPORATION 2002
```
Comment (sam-ellis) [inline src/IBM-pibs.xml]: Consider changing lines 15-18 into \<copyright> section.

Comment (jlovejoy) [inline src/IBM-pibs.xml]: I think this is another "vanity" licenses where a name appears throughout, so not marking up copyright is okay (and we didn't decide to mark it up when we added it.

Comment (bradleeedmondson) [inline src/IBM-pibs.xml]: no strong feeling on this; what would be the harm of marking lines 15 and 16 with the copyright tag?

Comment (pmadick) [inline src/IBM-pibs.xml]: Agreed that we don't need copyright tag.

**Pull request merged**